<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Computing topological data</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonInputFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Input image file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInputFile">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCut">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Invert</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>371</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCompress">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Compress the output file?</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_cmd">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>255</y>
      <width>481</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOutputFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Inverted image file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputFile">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonInputFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>inputFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCut</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>inverter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOutputFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>outputFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>inputFileSelecter()</slot>
  <slot>outputFileSelecter()</slot>
  <slot>inverter()</slot>
 </slots>
</ui>
